docs/UserGuide.adoc

```diff
@@ -158,68 +173,70 @@ You can find a contact added to the address book easily with a name.
 
 Format: `find n\[name]`
 
-[NOTE]
-`[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-
-Example: +
-`find n\john DOE`
-
-==== Add Picture to Contact: `addpic`
-You can add a profile picture to a contact in your address book.
-
-Format: `addpic n\[name] pa\[path]`
-
 [NOTE]
 ====
 * `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-* Absolute file path is used for `[path]`.
+* You can find partial name
 ====
 
-Example: +
-`addpic n\john doe pa\C:\Users\aloha\Desktop\aloha.png`
+Examples:
 
-Sets the profile picture of John Doe to aloha.png.
+* `find n\john DOE` +
+Finds a saved contact named John Doe.
 
-==== Update Picture to Contact: `updatepic`
-You can update the profile picture of a contact in your address book.
+* `find n\john` +
+Finds a saved contact named John.
```

Comment:
Find just one John or all contacts with John inside their name? John Tan, John Lim, John DOE? I don't know why you would know some many Johns haha. Sorry getting tired.